agent/php_txn.c

```diff
@@ -913,17 +915,20 @@ nr_status_t nr_php_txn_begin(const char* appnames,
     nr_php_txn_log_error_dt_on_tt_off();
   }
 
-#if ZEND_MODULE_API_NO >= ZEND_8_1_X_API_NO
-  if (nr_php_ini_setting_is_set_by_user("opcache.enable")
-      && NR_PHP_PROCESS_GLOBALS(preload_framework_library_detection)) {
-    nr_php_user_instrumentation_from_opcache(TSRMLS_C);
-  }
-#else
-  if (nr_php_ini_setting_is_set_by_user("opcache.preload")
-      && NR_PHP_PROCESS_GLOBALS(preload_framework_library_detection)) {
-    nr_php_user_instrumentation_from_opcache(TSRMLS_C);
+  /*
+   * Only try to instrument preloaded opcache scripts when opcache enabled and
+   * preload is not null.  If an INI value does not exist, INI_INT/INI_BOOL
+   * returns 0 and INI_STR returns NULL.
+   */
+  if (NR_PHP_PROCESS_GLOBALS(preload_framework_library_detection)) {
+    bool opcache_enabled = (0 != NR_PHP_PROCESS_GLOBALS(cli))
```

Comment:
Small nit but there is `bool is_cli = (0 != NR_PHP_PROCESS_GLOBALS(cli));` already defined earlier in the file [here](https://github.com/zsistla/newrelic-php-agent/blob/opcache_checks/agent/php_txn.c#L662). Consider re-using it here:
```suggestion
    bool opcache_enabled = is_cli
```